server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/RSpamDScanner.java

```
@@ -0,0 +1,116 @@
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.transport.mailets;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.List;
+import java.util.Optional;
+
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+
+import org.apache.james.core.MailAddress;
+import org.apache.james.rspamd.client.RSpamDClientConfiguration;
+import org.apache.james.rspamd.client.RSpamDHttpClient;
+import org.apache.james.rspamd.model.AnalysisResult;
+import org.apache.mailet.Attribute;
+import org.apache.mailet.AttributeName;
+import org.apache.mailet.AttributeValue;
+import org.apache.mailet.Mail;
+import org.apache.mailet.PerRecipientHeaders;
+import org.apache.mailet.base.GenericMailet;
+
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
+import com.google.common.collect.ImmutableList;
+
+public class RSpamDScanner extends GenericMailet {
+    public static final AttributeName FLAG_MAIL = AttributeName.of("org.apache.james.rspamd.flag");
+    public static final AttributeName STATUS_MAIL = AttributeName.of("org.apache.james.rspamd.status");
+    static final String RSPAMD_URL = "rSpamDUrl";
+    static final String RSPAMD_PASSWORD = "rSpamDPassword";
+    static final String RSPAMD_TIMEOUT = "rSpamDTimeout";
+
+    private RSpamDHttpClient rSpamDHttpClient;
+
+    @Override
+    public void init() throws MessagingException {
+        Optional<Integer> rSpamDTimeoutConfigure = Optional.ofNullable(getInitParameter(RSPAMD_TIMEOUT))
+            .filter(s -> !s.isEmpty())
+            .map(Integer::valueOf)
+            .filter(i -> i > 0);
```

Instead we should explicitly fail when the value is invalid eg `-1`!